pyplanet/apps/contrib/brawl_match/__init__.py

```diff
@@ -199,14 +207,19 @@ def __init__(self, *args, **kwargs):
 		if not self.match_tasks:
 			await self.brawl_chat(f'No match is currently in progress!', player)
 			return
+
+		await self.brawl_chat(f'Admin {player.nickname}$z$fff stopped match!')
 		for task in self.match_tasks:
 			if not task.done():
 				task.cancel()
 		self.match_tasks = []
-		await self.brawl_chat(f'Admin {player.nickname}$z$fff stopped match!')
+
+		map(lambda v: await v.destroy, self.views_open)
```

Comment:
`(await view.destroy) for view in self.view_open`
Should work too I'm pretty sure. But test it first to make sure.

Reply:
That syntax is not allowed it seems :D

Reply:
Then i would use a full for loop. IE `for view in self.views_open: view.destroy()`

I know im quite anti `map` here but i really dont like mapping to a lambda function.